### PR TITLE
Documentation for configure development environment

### PR DESCRIPTION
**Pull request by vstavetski**
add docs for configure and run markets dev environment

### CODE REVIEW

File: dev-environment/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+```bash
+docker-compose -f dev-environment/env.yml up -d
+```
+### Bootstrap Elassandra with keyspaces(required)
+```bash
+docker cp dev-environment/elassandra-bootstrap.cql  elassandra-markets:/elassandra-bootstrap.cql
+docker exec -it elassandra-markets bash
+cqlsh -f elassandra-bootstrap.cql
+```
+Then copy paste all content from elassandra-bootstrap.cql to interactive console and press enter. Write twice exit to
```
Comment (hleb-albau) [inline dev-environment/README.md]: remove 24-25 lines, also need fix this in search too.

File: dev-environment/env.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+  fast-data-dev:
+    container_name: fast-data-dev-markets
+    image: landoop/fast-data-dev:latest
+    ports:
+      - "3030:3030"
+      - "9092:9092"
+      - "2181:2181"
+      - "9581-9584:9581-9584"
+      - "8081-8083:8081-8083"
```
Comment (hleb-albau) [inline dev-environment/env.yml]: do not expose 8080-8081

File: dev-environment/README.md (hunk 1 above)
Comment (vstavetski) [reply dev-environment/README.md]: removed

File: dev-environment/env-mac.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+  fast-data-dev:
+    container_name: fast-data-dev-markets
+    image: landoop/fast-data-dev:latest
+    ports:
+      - "3030:3030"
+      - "9092:9092"
+      - "2181:2181"
+      - "9581-9584:9581-9584"
+      - "8081-8083:8081-8083"
```
[the remark this answers is not in the document]
Comment (vstavetski) [reply dev-environment/env-mac.yml]: done

File: dev-environment/env.yml (hunk 2 above)
Comment (vstavetski) [reply dev-environment/env.yml]: done

File: dev-environment/prometheus.yml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+global:
+  scrape_interval: 5s
```
[the remark this answers is not in the document]
Comment (vstavetski) [reply dev-environment/prometheus.yml]: increased to 15 seconds

**Pull request merged**